src/App.js

```diff
@@ -56,14 +65,15 @@ class App extends React.Component {
   }
 
   inputKeyPress = (e) => {
-    var keycode = (e.keyCode ? e.keyCode : e.which);
+    var keycode = e.keyCode ? e.keyCode : e.which;
     if (keycode === 13){
       this.addTodo();
     }
 
   }
   
   render() {
+    this.saveTodos();
     return (
       <div>
         <TodoList
```

Comment:
- Amazing  job on the assignment Arron. 
- You are properly handling state and all the events in React. 
- You split Your app into smaller components to make them modular. Your todo app is fully functional and it is possible to add new tasks, mark them as complete and remove them.
- For today you just have to styled it 👍 🎖 
- Your git workflow is really fluid and detailed, keep up the good job with this.